addon-test-support/ember-mocha/ember-test-context.js

```diff
@@ -0,0 +1,43 @@
+export default class EmberTestContext {
+  constructor(mochaContext) {
+    this.element = null;
```

Comment:
should this only apply to rendering tests?

Reply:
That had been my initial thought, but the `visit()` method, used in acceptance tests, [assigns `context.element`](https://github.com/emberjs/ember-test-helpers/blob/fdcf31ab04b908e4561eeea102fc07a6f483d640/addon-test-support/%40ember/test-helpers/setup-application-context.ts#L127-L131) without it having been set up on the context beforehand. I suppose we could assign `element` to the context in `setupApplicationTest()` (it already gets assigned as part of the call to `setupRenderingContext()` in `setupRenderingTest()`) instead of declaring it here, so that it's more explicit why it's needed. 🤔